api/holiday_acres_api/views.py

```diff
@@ -1,3 +1,4 @@
+from asyncio.windows_events import NULL
```

Comment:
There a reason you're importing this? Don't see it being used below.

Reply:
...I think it was part of a solution I copy/pasted, but yeah it doesn't do anything. Out it goes.